src/main/java/frc/robot/subsystems/Swerve/SwerveSubsystem.java

```diff
@@ -1,47 +1,51 @@
 package frc.robot.subsystems.Swerve;
 
 import static frc.robot.Constants.Constants.*;
-
-import java.util.Optional;
-
-import edu.wpi.first.math.VecBuilder;
-import edu.wpi.first.math.estimator.SwerveDrivePoseEstimator;
-
+import static frc.robot.RobotContainer.POSE_ESTIMATOR;
+import org.littletonrobotics.junction.Logger;
 import edu.wpi.first.math.geometry.Translation2d;
 import edu.wpi.first.math.kinematics.ChassisSpeeds;
 import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
 import edu.wpi.first.math.kinematics.SwerveModulePosition;
 import edu.wpi.first.math.kinematics.SwerveModuleState;
-import edu.wpi.first.networktables.GenericEntry;
-import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
-import edu.wpi.first.wpilibj.smartdashboard.Field2d;
+import edu.wpi.first.wpilibj.shuffleboard.SimpleWidget;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.LimelightHelpers;
 import frc.robot.Constants.SwerveModuleConfiguration;
 
-import frc.robot.subsystems.IMU;
-import edu.wpi.first.math.geometry.Pose2d;
-import edu.wpi.first.math.geometry.Rotation2d;
-
-
-
+import java.util.Map;
 
 public class SwerveSubsystem extends SubsystemBase {
-	
-	
-	private GenericEntry currentPos = Shuffleboard.getTab("swerve").add("curr_pos",new double[3]).getEntry();
-	
-	
-	private IMU imu = new IMU();
-
-	public SwerveDrivePoseEstimator pose_est;
-
-	Field2d field = new Field2d();
-	
+	public void velocityGraphUpdate(double xVelocity, double yVelocity){
+		SmartDashboard.putNumber("xVelocity Graph", xVelocity);
+		SmartDashboard.putNumber("yVelocity Graph", yVelocity);
+	}
+	private SimpleWidget speedMultiplierWidget = Shuffleboard.getTab("Drive")
+			.add("Max Speed", .5)
+			.withWidget(BuiltInWidgets.kNumberSlider)
+			.withProperties(Map.of("min", 0, "max", 1));
+
+	private SimpleWidget angularMultiplierWidget = Shuffleboard.getTab("Drive")
+			.add("Max Angular Speed", .5)
+			.withWidget(BuiltInWidgets.kNumberSlider)
+			.withProperties(Map.of("min", 0, "max", 1));
+	// ---------------WIP------------------
```

Comment:
Please remove commented code.

Reply:
@AreaProgram I believe you were working on this code, can you either commit it or remove it?

Reply:
Yeah, ill remove it for now.